index.js

```diff
@@ -4,13 +4,61 @@ var pluralize = require("pluralize");
 module.exports = function(app, handler){
 	return {
 		addResource : function(name){
-			app.use("/" + pluralize(name), function(req, res, next){
-				handler.findAll(S("_" + name).camelize().s)
-				.then(function(records){
-					res.json(records);
-				})
-				.catch(next);
-			});
+			var resourceName = S("_" + name).camelize().s;
+
+			var buildRoutes = function(name){
+				var router = require("express").Router();
+
+				router.param("id", function(req, res, next, id){
```

Comment:
They are isolated in the latest version: 

- Param callback functions are local to the router on which they are defined. They are not inherited by mounted apps or routers. Hence, param callbacks defined on router will be triggered only by route parameters defined on router routes.